clang/lib/CodeGen/CGOpenMPRuntimeGPU.cpp

```diff
@@ -3362,6 +3362,7 @@ Address CGOpenMPRuntimeGPU::getAddressOfLocalVariable(CodeGenFunction &CGF,
       break;
     case OMPAllocateDeclAttr::OMPLargeCapMemAlloc:
     case OMPAllocateDeclAttr::OMPCGroupMemAlloc:
+      AS = getLangASFromTargetAS(CGF.CGM.getModule().getDataLayout().getDefaultGlobalsAddressSpace());
```

Comment:
This line is way too long compared to the 80 char line limit

Reply:
ah I see, what would be the best way to update that line? Push a commit or create another PR?

Reply:
Running clang-format on your commit (but just your commit, and only accepting changes to the lines you've changed, i.e. just this line) to fix this doesn't need pre-commit review

Reply:
thanks I updated it